openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -3676,6 +3677,17 @@ def create_force(self, system, topology, **kwargs):
                 force.setCutoffDistance(to_openmm(self.cutoff))
                 force.setEwaldErrorTolerance(1.0e-4)
 
+                if self.switch_width >= self.cutoff:
```

Comment:
This case .... doesn't make much as much sense once the interpretation of `switch_width` matches up with the intended meaning. I'll have to update this logic and tests.